cmd/getConfig.go

```diff
@@ -57,7 +57,10 @@ func generateConfig() error {
 
 	customConfig, err := configOptions.openFile(GeneralConfig.CustomConfig)
 	if err != nil {
-		return errors.Wrapf(err, "config: open configuration file '%v' failed", GeneralConfig.CustomConfig)
+		if errors.Unwrap(err) != syscall.ENOENT {
```

Comment:
In which conditions will this be true? so, what are "legitimate" error cases?

Reply:
If the root error is syscall.ENOENT, ie. the file does not exist (NO ENTRY).

Reply:
I.e. we ignore only that error in particular.

Reply:
legitimate error cases would be stuff like ENOPERM, or whatever else can come back from open().

Reply:
Yes, but I can't think of examples for that. Can you provide examples?

Reply:
shouldn't we rather use `os.IsNotExist(err)` ?

Reply:
Yes, I didn't know it existed. (no pun intended)

Reply:
In any case, the behavior of the previous code was to check if the file exists, any only then try to open it. So if I just want to  re-instate the previous behavior, I think I can do that by checking only for this specific error from open().